by_Owership.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+Hosp_info=pd.read_csv('/home/iceboy/Documents/dataframe/HospInfo.csv')
+def Data_clean():
+    delt = Hosp_info[(Hosp_info['Hospital_rating']=='Not Available') | (Hosp_info['Efficient use of medical imaging national comparison']=='Not Available') | (Hosp_info['Patient experience national comparison']=='Not Available') | (Hosp_info['Mortality national comparison']=="Not Available") | (Hosp_info['Safety of care national comparison']=="Not Available")].index
+    Hosp_info.drop(delt, inplace=True)
+    Hosp_info.drop(["Timeliness of care national comparison footnote","Hospital overall rating footnote","Mortality national comparison footnote","Safety of care national comparison footnote","Readmission national comparison footnote","Patient experience national comparison footnote","Effectiveness of care national comparison footnote","Timeliness of care national comparison footnote","Efficient use of medical imaging national comparison footnote","Location"],axis = 1,inplace=True)
+def By_Types(type):
+    Types=Hosp_info[Hosp_info.Hospital_Ownership==type][['Hospital Name','Hospital Type','Hospital_rating','Phone Number','Address','State']]
+    print(Types)
+    draw_graph()
+def draw_graph():
+    State_wise=Hosp_info.groupby('Hospital_Ownership').count()[['Provider ID']].plot(kind='bar')
+    plt.show()
+if __name__ == "__main__":
+    Data_clean()
+    type=int(input("Enter 1 for Government \n 2 for Propietary \n 3 for voluntary(church) \n 4 for Voluntary(Private) \n 5 for Voluntary(others) \n 6 for Physician \n 7 for Govt(local) \n 8 for Govt(state) \n 9 for Tribal \n 10 for Govt(Federal)    :"))
+    if type==1:
```

Comment:
`type` is a python[ inbuilt class](https://docs.python.org/3/library/functions.html#type).
It's python pep8 standard to not use the name of python inbuilt classes for variable or function names.
you can use `_type` instead.

Reply:
Okay will modify it.